conf/application.conf

```diff
@@ -92,8 +92,8 @@ microservice {
     cubiks-gateway {
       url = "http://localhost:9288"
       scheduleIds {
-        standard = 15547
-        gis = 15593
+        standard = [ 15547, 12345 ]
```

Comment:
Because etray is also going to be a Cubiks test there should probably something that marks these as 'phase1' here - phase1 { standard []  gis [] } or phase1-standard perhaps?